mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1283,7 +1283,7 @@ class BinaryLogisticRegressionSummary private[classification] (
  *    P(y_i=K-1|\vec{x}_i, \beta) = \frac{e^{\vec{x}_i^T \vec{\beta}_{K-1}}\,}{\sum_{k=0}^{K-1}
  *       e^{\vec{x}_i^T \vec{\beta}_k}}
  *    $$
- * </blockquote></p>
+ * </blockquote>
```

Comment:
BTW, they are still printed fine.

I manually changed the access modifier of the same instance with this and manually tested with `jekyll build`.

With the codes:

![2016-11-22 5 41 37](https://cloud.githubusercontent.com/assets/6477701/20591823/d70112d0-b26d-11e6-8079-a9599096e693.png)


**Before**

![2016-11-22 5 43 43](https://cloud.githubusercontent.com/assets/6477701/20591839/e79a8a86-b26d-11e6-9645-532740aa7900.png)

**After**

![2016-11-22 5 51 24](https://cloud.githubusercontent.com/assets/6477701/20591842/eb6e044e-b26d-11e6-9bcf-4b66e58e8ff1.png)

They are identical.